plugins/http/client_intercepter.go

```diff
@@ -30,12 +30,18 @@ type ClientInterceptor struct {
 
 func (h *ClientInterceptor) BeforeInvoke(invocation operator.Invocation) error {
 	request := invocation.Args()[0].(*http.Request)
-	s, err := tracing.CreateExitSpan(fmt.Sprintf("%s:%s", request.Method, request.URL.Path), request.Host, func(headerKey, headerValue string) error {
+	host := request.Host
+	if host == "" {
+		if request.URL != nil {
```

Comment:
```suggestion
	if host == "" && request.URL != nil {
```
I think it can be simplified to a single line.